vue/components/ui/molecules/search-persistent-local-storage/search-persistent-local-storage.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div class="search-persistent-local-storage">
+        <global />

```

Comment:
Why is this needed?

Reply:
Ok but that only means that whoever uses this component must include global. If we have a global in every component that defeats the purpose.

Move this to the story instead.

Reply:
Right, Got it ! 👍